acl/src/main/java/org/apache/rocketmq/acl/plain/PlainPermissionManager.java

```diff
@@ -393,6 +399,401 @@ public boolean updateAccessConfig(PlainAccessConfig plainAccessConfig) {
+        PlainAccessResource updatePlainAccessResource = new PlainAccessResource();
+        if (updateAccountMap.containsKey(AclConstants.CONFIG_ACCESS_KEY)) {
+            updatePlainAccessResource.setAccessKey(updateAccountMap.get(AclConstants.CONFIG_ACCESS_KEY).toString());
+        }
+        if (updateAccountMap.containsKey(AclConstants.CONFIG_SECRET_KEY)) {
+            updatePlainAccessResource.setSecretKey(updateAccountMap.get(AclConstants.CONFIG_SECRET_KEY).toString());
+        }
+        if (updateAccountMap.containsKey(AclConstants.CONFIG_WHITE_ADDR)) {
+            updatePlainAccessResource.setRemoteAddressStrategy(remoteAddressStrategyFactory.
+                getRemoteAddressStrategy(updateAccountMap.get(AclConstants.CONFIG_WHITE_ADDR).toString()));
+        }
+        if (updateAccountMap.containsKey(AclConstants.CONFIG_ADMIN_ROLE)) {
+            updatePlainAccessResource.setAdmin(Boolean.parseBoolean(updateAccountMap.get(AclConstants.CONFIG_ADMIN_ROLE).toString()));
+        }
+        if (updateAccountMap.containsKey(AclConstants.CONFIG_DEFAULT_TOPIC_PERM)) {
+            updatePlainAccessResource.setDefaultTopicPerm(Permission.
+                parsePermFromString(updateAccountMap.get(AclConstants.CONFIG_DEFAULT_TOPIC_PERM).toString()));
+        }
+        if (updateAccountMap.containsKey(AclConstants.CONFIG_DEFAULT_GROUP_PERM)) {
+            updatePlainAccessResource.setDefaultGroupPerm(Permission.
+                parsePermFromString(updateAccountMap.get(AclConstants.CONFIG_DEFAULT_GROUP_PERM).toString()));
+        }
+        if (updateAccountMap.containsKey(AclConstants.CONFIG_RESOURCE_PERMS)) {
+            List<LinkedHashMap<String, Object>> resourceAndPermList = (List<LinkedHashMap<String, Object>>)updateAccountMap.get(AclConstants.CONFIG_RESOURCE_PERMS);
+            for (LinkedHashMap resourceAndPerm : resourceAndPermList) {
+                String resource = resourceAndPerm.get("resource").toString();
+                String namespace = resourceAndPerm.get("namespace").toString();
+                String type = resourceAndPerm.get("type").toString();
+                String perm = resourceAndPerm.get("perm").toString();
+                if (namespace == null || namespace.isEmpty()) {
+                    if (type.equals(ResourceType.GROUP.toString())) {
+                        updatePlainAccessResource.addResourceAndPerm(MixAll.getRetryTopic(resource), Permission.parsePermFromString(perm));
+                    } else {
+                        updatePlainAccessResource.addResourceAndPerm(resource, Permission.parsePermFromString(perm));
+                    }
+                } else {
+                    if (type.equals(ResourceType.GROUP.toString())) {
+                        updatePlainAccessResource.addResourceAndPerm(
+                            MixAll.RETRY_GROUP_TOPIC_PREFIX + namespace + NAMESPACE_SEPARATOR + resource, Permission.parsePermFromString(perm));
```

Comment:
Make use of methods provided in `NamespaceUtil` to construct the resource. Do not import the concrete implementation of namespace formate in ACL module.